### ISSUE TITLE
[Stats] InflateException: Binary XML file line #6: Error inflating class ImageView

### ISSUE THREAD

**Issue by oguzkocer** [wordpress-mobile/WordPress-Android#12396]
**Sentry Url**: https://sentry.io/share/issue/77e4176af0794667aed2971cefb17921/
**User Count**: 4
**Count**: 23
**First Release**: org.wordpress.android@15.2-rc-1+886
**First Seen**: 2020-06-30T13:12:53.394000Z
**Last Seen**: 2020-07-09T05:04:32Z
**24 Hours**: 20
**30 Days**: 23

```
InflateException: Binary XML file line #6: Error inflating class ImageView
     at org.wordpress.android.ui.stats.refresh.lists.sections.viewholders.BlockListItemViewHolder.<init>(BlockListItemViewHolder.kt:25)
     at org.wordpress.android.ui.stats.refresh.lists.sections.viewholders.ImageItemViewHolder.<init>(ImageItemViewHolder.kt:9)
     at org.wordpress.android.ui.stats.refresh.lists.sections.BlockListAdapter.onCreateViewHolder(BlockListAdapter.kt:105)
     at org.wordpress.android.ui.stats.refresh.lists.sections.BlockListAdapter.onCreateViewHolder(BlockListAdapter.kt:87)
...
(106 additional frame(s) were not displayed)
```

```
Resources$NotFoundException

File res/drawable/insights_management_feature_image.xml from drawable resource ID #0x7f080208
```

**Comment by bummytime** [comment]
The sentry URL didn't work for me above ☝️ — updated URL: https://sentry.io/share/issue/7dae5c99df0f45ce89e09fa2335841c0/

Looks like this issue has been happening since Apr 24, 2020 and affects about 47 users. @frosty Any ideas on what's up with the missing `insights_management_feature_image.xml` resource referenced here??

**Comment by planarvoid** [comment]
I've investigated this crash and I think it deserves a hotfix. It's caused by older devices not being able to load a new vector drawable in the `Insights management News card`. This is not causing more issues because you have to have an old device and be a new user of the app.

**Comment by oguzkocer** [comment]
Fixed in #12451.
